build-system/common/utils.js

```diff
@@ -48,7 +50,48 @@ function getFilesChanged(globs) {
   });
 }
 
+/**
+ * Logs the list of files that will be checked.
+ *
+ * @param {!Array<string>} files
+ */
+function logFiles(files) {
+  if (!isTravisBuild()) {
+    log(green('INFO: ') + 'Checking the following files:');
+    files.forEach(file => {
+      log(cyan(file));
+    });
+  }
+}
+
+/**
+ * Gets a list of files to be checked based on command line args and the given
+ * file matching globs. Used by tasks like prettify, check-links, etc.
+ *
+ * @param {!Array<string>} globs
+ * @param {Object=} options
+ * @return {!Array<string>}
+ */
+function getFilesToCheck(globs, options = {}) {
+  let filesToCheck = [];
```

Comment:
nit: by returning early from every conditional block you can avoid mutating a variable. Otherwise it's also not needed to initialize the current `let` decl.

Reply:
Agreed. Rewrote this function.